book/getting-started/run-a-wallaroo-application.md

```diff
@@ -131,11 +131,19 @@ You can then click into one of the elements within a category, to get to a detai
 
 Feel free to click around and get a feel for how the Metrics UI is setup and how it is used to monitor a running Wallaroo application. If you'd like a deeper dive into the Metrics UI, have a look at our [Monitoring Metrics with the Monitoring Hub](/book/metrics/metrics-ui.md) section.
 
-## Shutting Down The Cluster
+## Shutdown
 
 You can shut down the cluster with this command once processing has finished:
 
 ```bash
 cd ~/wallaroo-tutorial/wallaroo/utils/cluster_shutdown
 ./cluster_shutdown 127.0.0.1:5050
 ```
+
+You can shut down Giles Sender and Giles Receiver by pressing Ctrl-c from their respective shells.
+
+You can shut down the Metrics UI with the following command:
+
+```bash
+docker stop mui
```

Comment:
Should we also include a note about removing the docker container for the metrics UI?

Reply:
@dipinhora we previously mention using `docker start mui` to start it up if it's been shut down. Any good reason to remove it entirely?

Reply:
@dipinhora I'm leaning toward "no" because I think writing too much about how to use 3rd party tools just adds noise to our documentation. I realize that I'm drawing an arbitrary line at how much is too much.

Reply:
Works for me.